### ISSUE TITLE
Bad date generated for Solidity's 'now'

### ISSUE THREAD

**Issue by maurycyp** [pipermerriam/eth-testrpc#25]
When my Solidity contracts store a date with `now`, I later get a value like 1410973360 when I query that date.  This is about a year in the past (the clock on my computer is fine).  Perhaps this is a pyethereum issue?

**Comment by Smithgift** [comment]
I am experiencing this as well.

**Comment by Smithgift** [comment]
More information: it seems that time passes freakishly fast. Take this:
```
contract TimeTest {
	uint public lastTime;
	function TimeTest() {
		lastTime = now;	
	}
	function timeDiff() constant returns (uint) {
		return (now - lastTime);
	}
}
```
Create a contract and call timeDiff() a few times in a row, and see how fast time passes. This seems to be intentional--is it?

**Comment by tcoulter** [comment]
@Smithgift I'm not entirely sure. Unfortunately, I'm no longer actively maintaining TestRPC. Instead, I've moved to EtherSim as it's easier for me to develop in, and is more straightforward (EtherSIM talks to the underlying VM directly, rather than through a "tester" layer). 

I maintain my own version of EtherSim here: https://github.com/tcoulter/Ethersim . Since EtherSim's release my colleagues and I have committed the bulk of the new features. This version of EtherSim, until stated otherwise, is the one most likely to work with Truffle.

Cheers, and hopefully that helps!
